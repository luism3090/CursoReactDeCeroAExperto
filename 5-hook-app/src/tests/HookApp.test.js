import React from 'react';
import {shallow} from 'enzyme';
import '@testing-library/jest-dom';
import { HookApp } from "../HookApp";

describe('Validar el archivo Hook App', () => {
    
    test('debe de hacer el snapchoop del HookApp ', () => {

        const wrapper = shallow(<HookApp />);

        expect(wrapper).toMatchSnapshot();

    });

    

});
