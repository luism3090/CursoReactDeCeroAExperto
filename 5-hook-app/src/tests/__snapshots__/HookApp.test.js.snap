// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validar el archivo Hook App debe de hacer el snapchoop del HookApp  1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
